### PR TITLE
Amendments #5-7

### PR DESCRIPTION
**Pull request by thatging3rkid**
On April 21st, 2020, amendments 5-7 (Miscellaneous Changes \#1, Changes to Decision Making Process, Addition of Diminished Club State) were formally proposed to the club at-large.

### CODE REVIEW

**Comment by thatging3rkid** [comment]
Draft amendment proposal v1.0: 
[amendment.pdf](https://github.com/RITFIRSTRobotics/constitution/files/4507656/amendment.pdf)

**Comment by thatging3rkid** [comment]
Draft amendment proposal v1.1:
[amendment.pdf](https://github.com/RITFIRSTRobotics/constitution/files/4513415/amendment.pdf)

**Review by ahkneipp — APPROVED** [review]
Looks good to me

**Comment by thatging3rkid** [comment]
Voted and passed on April 28th/29th 2020, awaiting advisor approval

**Comment by thatging3rkid** [comment]
Had to add some things to be compliant with the Fall 2019 Center for Campus Life constitution requirements, should be all set now, just waiting for a review then I'll merge it.

File: constitution.tex
Comment (ahkneipp) [inline constitution.tex]: Typo

Comment (ahkneipp) [inline constitution.tex]: Should have an oxford comma

**Review by ahkneipp — COMMENTED** [review]
Typos and grammar

File: constitution.tex
Comment (thatging3rkid) [reply constitution.tex]: Both these grammar issues are from the CCL documents that said to copy verbatim, so...

Fixed them anyways, should be all set.

**Review by ahkneipp — APPROVED** [review]

**Pull request merged**
